tests/integ/test_pandas_to_df.py

```diff
@@ -248,9 +309,7 @@ def test_write_pandas_temp_table_and_irregular_column_names(session):
     )
     table_name = Utils.random_name_for_temp_object(TempObjectType.TABLE)
     try:
-        session.write_pandas(
-            pd, table_name, auto_create_table=True, create_temp_table=True
-        )
+        session.write_pandas(pd, table_name, auto_create_table=True, table_type="temp")
```

Comment:
same point. does making `table_type="transient"` work?

Reply:
Oh I overlooked this and thought the usage of temp table is just for test purpose. I'll make it parametrized and cover transient as well.